tensorflow/compiler/jit/flags.cc

```diff
@@ -65,7 +74,9 @@ void AppendMarkForCompilationPassFlagsInternal(std::vector<Flag>* flag_list) {
       Flag("tf_xla_auto_jit", SetterForXlaAutoJitFlag, "0",
            "Control compilation of operators into XLA computations on CPU and "
            "GPU devices.  0 = use ConfigProto setting; -1 = off; 1 = on for "
-           "things very likely to be improved; 2 = on for everything.  "
+           "things very likely to be improved; 2 = on for everything; "
+           "(experimental) fusible = only for Tensorflow operations that XLA "
```

Comment:
Do we actually need a separate mode here?
I don't like "fusible" too much because it's not consistent with previously numeric values, and I also wonder why is it not sufficient to set `--tf_xla_auto_jit=2 --tf_xla_ops_to_cluster=FUSIBLE` (we can also have `--tf_xla_ops_to_cluster=FUSIBLE` imply `--tf_xla_auto_jit=2`). 

I would also like this mode to be a separate flag for now.

Reply:
TF already have tons of flags. This make them more difficult to discover and be used by users. Reusing existing flags help user find, use them and remember them. This is why we prefer to have a shortcut with existing flags. Changing the name of the new flag value to something is not a problem. Any suggestion? "lite"(as the project is called XLALite)?
This option heavily influence the auto_jit, so it is highly related to this parameter. Ideally, I would have created a new number like 1 and bumped the current value 1 and 2. But this would break the interface which isn't great.

Currently, --tf_xla_auto_jit=fusible imply --tf_xla_auto_jit=1, not =2. It is  not clear what the difference is between them for me and I didn't found the code which make a difference in which nodes get selected or not. All benchmarks have been done with 1. Is it a problem to change the value from 1 to 2?

Reply:
I prefer using `--tf_xla_auto_jit` as Frédéric proposed.

George, Sanjoy, and I have a meeting tomorrow and we'll put this question on the agenda. We'll get back to you tomorrow, Frédéric.

Reply:
OK!